templates/api_doc/index.html.twig

```diff
@@ -384,7 +384,7 @@ GET https://{{ packagist_host }}/statistics.json
 
   <h4 id="list-security-advisories-by-package">{{ 'api_doc.list_security_advisory'|trans }}</h4>
 
-    <p>This endpoint provides a list of security advisories. Either a list of packages as query or request parameter or a timestamp as updatedSince query parameter need to be passed.</p>
+    <p>This endpoint provides a list of security advisories. A list of packages as query or request parameter needs to be passed. An optional timestamp as updatedSince parameter can also be passed.</p>
```

Comment:
This is not correct. Sending one of them is enough however, `updatedSince` is currently expected to be set as query parameter. See https://github.com/composer/packagist/blob/main/src/Controller/ApiController.php#L322

Reply:
Ahh, interesting. I'm not quite sure I understand why the code is making this distinction then... but in any case this PR does not address the problem, if indeed there is a problem. I'll close it.

Reply:
Could probably just add the functionality to handle `updatedSince` sent as POST as you expected. I guess this just wasn't added yet because nobody needed it so far.

Reply:
I don't need it either now that I understand how to add the var to a POST request. It's a bit confusing, but it works.